manifests/init.pp

```diff
@@ -287,7 +290,7 @@
 
     class { 'pulp':
       enable_ostree          => $enable_ostree,
-      enable_rpm             => $enable_yum,
+      enable_rpm             => $enable_yum and !$proxy_pulp_yum_to_pulpcore,
```

Comment:
Not really for this PR, but this made me think of it: we don't have conditional to disable crane if we proxy container content to Pulpcore.

Reply:
Yea, I think we just were leaning towards leave Crane alone till we can safely rip it out.